Assets/MixedRealityToolkit/_Core/Services/SpatialAwarenssSystem/MixedRealitySpatialAwarenessSystem.cs

```diff
@@ -6,13 +6,11 @@
 using Microsoft.MixedReality.Toolkit.Core.EventDatum.SpatialAwarenessSystem;
```

Comment:
There's a typo in the folder name: `SpatialAwarenssSystem`

Reply:
Thanks for catching that!